teamwork/templates/projects/partial_project_profile.html

```diff
@@ -54,7 +62,7 @@ <h5 class="widget-user-desc">{{project.tagline}}</h5>
     {% if forloop.last %}</div><div class="clearfix"></div>{% endif %}
 {% empty %}
   <div>
-  </h3>You are not in any projects</h3>
```

Comment:
This should be you since you are addressing the user

Reply:
Wait what do you mean, can you clarify please?

Reply:
The message appears to have been changed to They but in our version it is You and You is the correct version since it it meant to be addressing the user